### PR TITLE
case-insensitive command matching

### PR DESCRIPTION
**Pull request by philchuang**
/Create = /CrEaTe = /create, etc

### CODE REVIEW

File: JabbR/JabbR.csproj [hunk 1]
```diff
@@ -22,9 +22,9 @@
     <OldToolsVersion>4.0</OldToolsVersion>
     <UpgradeBackupLocation />
     <IISExpressSSLPort />
-    <IISExpressAnonymousAuthentication />
-    <IISExpressWindowsAuthentication />
-    <IISExpressUseClassicPipelineMode />
+    <IISExpressAnonymousAuthentication>enabled</IISExpressAnonymousAuthentication>
```
Comment (davidfowl) [inline JabbR/JabbR.csproj]: This shouldn't be in this change.

Comment (philchuang) [inline JabbR/JabbR.csproj]: yup, noticed that too late, will revert

File: JabbR/Commands/CommandManager.cs [hunk 2]
```diff
@@ -157,7 +157,7 @@ select new
             }
             else
             {
-                candidates = _commandCache.Keys.Where(comm => comm.StartsWith(commandName))
+                candidates = _commandCache.Keys.Where(comm => comm.StartsWith(commandName, StringComparison.CurrentCultureIgnoreCase))
```
Comment (davidfowl) [inline JabbR/Commands/CommandManager.cs]: OrdinalIgnoreCase

**Comment by samandmoore** [comment]
what @davidfowl said, then looks good to me

**Pull request merged**